packages/hooks/src/__tests__/useWalletTransactions.spec.tsx

```diff
@@ -6,13 +6,24 @@ import useWalletTransactions from '../useWalletTransactions';
 
 jest.mock('@deriv/api', () => ({
     ...jest.requireActual('@deriv/api'),
+    useCurrencyConfig: jest.fn(() => ({ getConfig: () => ({ code: 'USD', type: 'fiat' }) })),
     useFetch: jest.fn(),
     usePaginatedFetch: jest.fn(),
 }));
 
 const mockUseFetch = useFetch as jest.MockedFunction<typeof useFetch<'authorize'>>;
 const mockUsePaginatedFetch = usePaginatedFetch as jest.MockedFunction<typeof usePaginatedFetch<'statement'>>;
 
+const createWrapper = (mock: ReturnType<typeof mockStore>) => {
```

Comment:
created a wrapper in order to replace duplicated code

Reply:
The next step could be to move it to a separate file ;)
To deduplicate it across several files

Reply:
yeap @seva-deriv you are tottaly right! just because this is only hooks package, would like to discuss it first with @farzin-deriv how to properly do this
thanks for pointing on this @seva-deriv 🙏🏻

Reply:
Moving the function into a separate file and importing it around would solve the duplication issue but will create more problems as devs will start importing files across packages, We need to improve the DX instead.

We have discussed this many times now 😅 But unfortunately still it's not clear what's the plan and we are kinda stuck in the middle 👀

Long story short; We have created 2 main abstractions around stores and API in the codebase to address the main issues we have which enable devs to be able to write tests, After some time we observed that some methods are being duplicated again and again, Which ring the bell on how we can improve the tooling we have provided for the writing test.

What I had in mind was to use jest globals to provide these utilities in test files without devs needing to do anything, Propbely there are better ways, I'm not that familiar with testing, Maybe @mohsen-deriv can share some ideas as I remember he did some work on this subject before.

The problem right now is that we don't know what is the next step, The tests we have which were written using these abstractions have been labeled as useless tests which we need to get rid of and replace with integration testing. So it's a waste of resources to work on something we don't want or even focus on removing duplicated code in the test file that is going to go away.

cc: @chrisgodfrey-deriv @yashim-deriv @mohsen-deriv @nijil-deriv